pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -840,6 +841,36 @@ public String getServerMetadata(
     return segmentsMetadata;
   }
 
+  @GET
+  @Path("segments/{tableName}/metadataV2")
+  @Authorize(targetType = TargetType.TABLE, paramName = "tableName", action = Actions.Table.GET_METADATA)
+  @Produces(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Get the server metadata for all table segments",
+      notes = "Get the server metadata for all table segments")
+  public String getServerSegmentsMetadata(
```

Comment:
What is the difference between this one and the original one?

Reply:
It makes one call per server whereas `segments/{tableName}/metadata` makes 1 call per segment to fetch the metadata

Reply:
Am I mis-reading? Seems this one is exactly the same as the deprecated one